PennMobile/Home/Fitness/FitnessSettingsView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  FitnessSettingsView.swift
+//  PennMobile
+//
+//  Created by Jordan H on 5/13/23.
+//  Copyright © 2023 PennLabs. All rights reserved.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct FitnessSelectView: View {
+    @Binding var showFitnessSettings: Bool
+    @ObservedObject var favoritesList: FavoritesList
+    var rooms: [FitnessRoom]
+    @State var selections: [Int: Bool]
+
+    var body: some View {
+        VStack {
+            HStack {
+                // Button to cancel settings
+                Button {
+                    showFitnessSettings = false
+                } label: {
+                    Text("Cancel")
+                        .foregroundColor(.blue)
+                }
+                Spacer()
+                Text("Select Favorites")
+                Spacer()
+                // Button to save settings
+                Button {
+                    let favorites = selections.keys.filter { selections[$0] == true }
+                    FitnessRoom.setPreferences(for: favorites)
+                    favoritesList.favorites = favorites
+                    showFitnessSettings = false
+                } label: {
+                    Text("Save")
+                        .foregroundColor(.blue)
+                }
+            }
+            .padding([.top, .leading, .trailing])
+            List {
+                ForEach(rooms) { room in
+                    FitnessSelectRowView(room: room, isSelected: selections[room.id]!, action: { selections[room.id] = !selections[room.id]!
```

Comment:
I feel like we should be using an abridged version of this view (no hours or open label) or we should merge the two (add a star button to the right of each item in the central fitness tab).